desc/optimize/optimizer.py

```diff
@@ -349,29 +349,45 @@ def optimize(  # noqa: C901 - FIXME: simplify this
             things[ind].params_dict = params
 
         if verbose > 0:
-            print("Start of solver")
-            # need to check index of things bc things0 contains copies of
-            # things, so they are not the same exact Python objects
-            objective.print_value(
-                objective.x(*[things0[things.index(t)] for t in objective.things])
-            )
-            for con in constraints:
-                arg_inds_for_this_con = [
-                    things.index(t) for t in things if t in con.things
-                ]
-                args_for_this_con = [things0[ind] for ind in arg_inds_for_this_con]
-                con.print_value(*con.xs(*args_for_this_con))
-
-            print("End of solver")
-            objective.print_value(
-                objective.x(*[things[things.index(t)] for t in objective.things])
+            state_0 = [things0[things.index(t)] for t in objective.things]
+            state = [things[things.index(t)] for t in objective.things]
+
+            print_pretty = (
```

Comment:
Tbh, I don't like this. But you guys asked for the old print to remain. I guess we can get rid of this and always print the comparison version

Reply:
We meant, the old use of `print_value` should be the same, if I were to do `obj.print_value` with only one set of args. 

I think we all want the optimizer to by default print the comparisons in the way you've changed them to, so you don't need this conditional here